cmd/collector/config/export.go

```diff
@@ -176,3 +218,46 @@ func getLabelMappings(mappings []LabelMatcher) (map[*regexp.Regexp]*regexp.Regex
 	}
 	return regexMappings, nil
 }
+
+type FluentForwardLogExport struct {
+	Name         string `toml:"name" comment:"Name of the exporter."`
+	Destination  string `toml:"destination" comment:"Fluentd/Fluent Bit endpoint to send logs to. Must be in the form tcp://<host>:<port> or unix:///path/to/socket."`
+	TagAttribute string `toml:"tag-attribute" comment:"Attribute key to use as the tag for the log. If the attribute is not set, the log will be ignored by this exporter."`
+}
+
+func (f *FluentForwardLogExport) Validate() error {
+	if f.Name == "" {
+		return fmt.Errorf("name must be set")
+	}
+	if f.Destination == "" {
+		return fmt.Errorf("destination must be set")
+	}
+
+	if strings.HasPrefix(f.Destination, "unix://") {
+		socketPath := strings.TrimPrefix(f.Destination, "unix://")
+		if len(socketPath) == 0 {
+			return fmt.Errorf("invalid destination %s: unix socket path is empty", f.Destination)
+		}
```

Comment:
Should we verify the path exists and we have permissions to read from it?

```suggestion
		}
		fileInfo, err := os.Stat(path)
	        if err != nil {
		    if os.IsNotExist(err) {
			return fmt.Errorf("path does not exist: %s", path)
			}
	
	}
```

Reply:
I don't think so. We don't check file paths (or socket destinations) in our config validation right now. I think it's good that we don't so that we can more easily create a validate subcommand to check that a config is well-formed without needing to be on an actual host with all of the available paths, etc.